backend/Migrations/WhaleSpottingContextModelSnapshot.cs

```diff
@@ -374,22 +376,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .OnDelete(DeleteBehavior.Cascade)
                         .IsRequired();
                 });
-
-            modelBuilder.Entity("WhaleSpotting.Models.Data.Sighting", b =>
-                {
-                    b.HasOne("WhaleSpotting.Models.Data.Species", "Species")
-                        .WithMany("Sightings")
-                        .HasForeignKey("SpeciesId")
-                        .OnDelete(DeleteBehavior.Cascade)
-                        .IsRequired();
-
-                    b.Navigation("Species");
-                });
-
-            modelBuilder.Entity("WhaleSpotting.Models.Data.Species", b =>
-                {
-                    b.Navigation("Sightings");
-                });
```

Comment:
it looks like maybe something to do with the sightings and speciesId foreign key has been removed here - I assume that wasn't intentional and could be a bit to do with the 'magic' of entity framework core - do you want to have a look into that and see what happened/ if there is actually an issue because of this?